lib/js/components/DatePickers/DatePickerBox/DatePickerBox.vue

```diff
@@ -0,0 +1,278 @@
+			'-ds-isOpen': isOpen,
+		}"
+	>
+		<span v-if="eyebrowText" class="ds-datePickerBox__eyebrow">{{ eyebrowText }}</span>
+		<div class="ds-datePickerBox__dateWrapper">
+			<ds-icon
+				v-if="icon"
+				class="ds-datePickerBox__icon"
+				:class="{ '-ds-hiddenOnMobile': isIconHiddenOnMobile }"
+				:icon="icon"
+				:size="ICON_SIZES.XX_SMALL"
+			></ds-icon>
+			<span class="ds-datePickerBox__date">{{ text }}</span>
+		</div>
+
+		<div v-if="state === DATE_PICKER_STATES.LOADING" class="ds-datePickerBox__loader">
+			<ds-icon :icon="ICONS.FAD_SPINNER_THIRD" :size="ICON_SIZES.SMALL" spinning />
+		</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../../styles/settings/typography/tokens';
+@import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/spacings';
+@import '../../../../styles/settings/media-queries';
+@import '../../../../styles/settings/borders';
+@import '../../../../styles/settings/radiuses';
+
+@mixin color-scheme(
+	$color-eyebrow,
+	$color-date,
+	$color-icon,
+	$color-icon-hovered,
+	$color-date-hovered,
+	$color-icon-disabled,
+	$color-eyebrow-disabled,
+	$color-date-disabled,
+	$self
+) {
+	&.-ds-interactive {
+		&:hover {
+			#{$self}__icon {
+				color: $color-icon-hovered;
+			}
+
+			#{$self}__date {
+				color: $color-date-hovered;
+			}
+		}
+	}
+
+	&.-ds-disabled {
+		#{$self}__eyebrow {
+			color: $color-eyebrow-disabled;
+		}
+
+		#{$self}__date {
+			color: $color-date-disabled;
+		}
+
+		#{$self}__icon {
+			color: $color-icon-disabled;
+		}
+	}
+
+	&.-ds-isOpen {
+		&:hover {
+			#{$self}__icon {
+				color: $color-icon;
+			}
+
+			#{$self}__date {
+				color: $color-date;
+			}
+		}
+	}
+	#{$self}__eyebrow {
+		color: $color-eyebrow;
+	}
+
+	#{$self}__date {
+		color: $color-date;
+	}
+
+	#{$self}__icon {
+		color: $color-icon;
+	}
+
+	#{$self}__loader {
+		color: $color-icon;
+	}
+}
+
+.ds-datePickerBox {
+	$self: &;
+
+	border: $border-xs solid transparent;
+	display: flex;
+	flex-direction: column;
+	padding: $space-2xs $space-xs;
+	pointer-events: none;
+	position: relative;
+
+	&.-ds-interactive {
+		background-color: $color-neutral-background-weak;
+		border-color: $color-neutral-background;
+		border-radius: $radius-m;
+		cursor: pointer;
+		pointer-events: all;
+
+		&:hover {
+			background-color: $color-neutral-background-weak-hovered;
+		}
+	}
+
+	&.-ds-disabled {
+		background-color: $color-neutral-background-weak-disabled;
+		pointer-events: none;
+	}
+
+	&.-ds-isOpen {
+		background-color: $color-neutral-background-medium;
+
+		&:hover {
+			background-color: $color-neutral-background-medium;
+		}
+	}
+
+	&__eyebrow {
+		@include info-xs-default-regular;
+	}
+
+	&__dateWrapper {
+		column-gap: $space-4xs;
+		display: flex;
+	}
+
+	&__date {
+		@include text-s-default-regular;
+	}
+
+	&__icon {
+		&.-ds-hiddenOnMobile {
+			display: none;
+
+			@media (#{breakpoint-s()}) {
+				display: flex;
+			}
+		}
+	}
+
+	&__loader {
```

Comment:
![Zrzut ekranu 2024-07-1 o 13 28 50](https://github.com/bethinkpl/design-system/assets/4587534/d165f139-3a6b-4f56-8ac5-41542933a964)
loader is displayed incorrectly, in Figma the box is 76x44px, in DS it is 71x36 and it is not in the center

Reply:
the icon is too big too

Reply:
🚀